asr_automation/asr_automation_pipeline.ipynb

```diff
@@ -0,0 +1,347 @@
+      ],
+      "metadata": {
+        "id": "kbfXGIBhvJ6x"
+      },
+      "id": "kbfXGIBhvJ6x"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from superannotate import SAClient\n",
+        "\n",
+        "sa = SAClient(token=SDK_TOKEN)"
+      ],
+      "metadata": {
+        "id": "BqgWghXHvOQK"
+      },
+      "id": "BqgWghXHvOQK",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#3.2 Create the project and attach the dataset"
+      ],
+      "metadata": {
+        "id": "ba7slGyyvgHg"
+      },
+      "id": "ba7slGyyvgHg"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "721cdc12",
+      "metadata": {
+        "id": "721cdc12"
+      },
+      "outputs": [],
+      "source": [
+        "# 1. create a Video project\n",
+        "sa.create_project(PROJECT_NAME, \"---\", \"Video\")\n",
+        "\n",
+        "# 2. create dataset CSV to attach to the platform\n",
+        "import pandas as pd\n",
+        "\n",
+        "data = {\n",
+        "    \"name\": DATASET_FILES,\n",
+        "    \"url\": [join(S3_BASE_URL, dataset_file) for dataset_file in DATASET_FILES]\n",
+        "}\n",
+        "df = pd.DataFrame(data)\n",
+        "df.to_csv('data.csv')\n",
+        "\n",
+        "# 3. attach dataset\n",
+        "sa.attach_items(PROJECT_NAME, \"./data.csv\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "6ca3cef6",
+      "metadata": {
+        "id": "6ca3cef6"
+      },
+      "source": [
+        "## 4. Import Predictions"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#4.1 Construct the annotation dictionaries"
+      ],
+      "metadata": {
+        "id": "ZLitcBJYxENW"
+      },
+      "id": "ZLitcBJYxENW"
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "annotations = []\n",
+        "for file_name, transcribtion in transcribtions.items():\n",
+        "  sa_format = {\n",
+        "      \"metadata\": {\n",
+        "          \"name\": file_name\n",
+        "      },\n",
+        "      \"instances\": [],\n",
+        "      \"tags\": [transcribtion],\n",
+        "      \"comments\": []\n",
+        "  }\n",
+        "  annotations.append(sa_format)"
+      ],
+      "metadata": {
+        "id": "G5GA0Qn6xB5S"
+      },
+      "id": "G5GA0Qn6xB5S",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "#4.2 Upload the transcribtions to the platform"
+      ],
+      "metadata": {
+        "id": "ggoa8TZDxt8K"
+      },
+      "id": "ggoa8TZDxt8K"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "a34126fa",
+      "metadata": {
+        "id": "a34126fa"
+      },
+      "outputs": [],
+      "source": [
+        "sa.upload_annotations(PROJECT_NAME, annotations)"
```

Comment:
Let's put here some screenshots of how transcriptions look in SA Video Editor UI